WORKSPACE.bazel

```diff
@@ -76,6 +77,7 @@ canisters(
         "ck_eth_ledger_v2_noledgerversion": "mainnet_cketh_ic-icrc1-ledger-u256-v2-noledgerversion",
         "ck_eth_ledger_v3": "mainnet_cketh_ic-icrc1-ledger-u256-v3",
         "ck_eth_index": "mainnet_cketh-index-ng",
+        "cycles_ledger_index": "mainnet_cycles_ledger_index",
```

Comment:
why no -ng in the name?

Reply:
The actual canister file name is `ic-icrc1-index-ng-u256.wasm.gz` (i.e., `ng` as well as `u256`), but why would we add such a suffix given that there's only a single cycles ledger index deployed?